### DISCUSSION TITLE
No Artifacts in UI show Although log shows that Artifacts has been uploaded successfully

### DISCUSSION THREAD

**Discussion by Abirdcfly** [community/community#47721] [Actions]
### Select Topic Area

Bug

### Body

github action is this https://github.com/bestchains/fabric-operator/actions/runs/4193043025/attempts/5

![CleanShot 2023-02-17 at 10 03 20](https://user-images.githubusercontent.com/5100555/219530737-2ec45c14-ce02-4155-8f74-9ba76670a9e4.png)

And github log show:
https://github.com/bestchains/fabric-operator/actions/runs/4193043025/jobs/7283530966#step:7:23
![CleanShot 2023-02-17 at 10 04 10](https://user-images.githubusercontent.com/5100555/219530862-7f297017-a4ab-43a8-a295-30ff1cffd160.png)

Maybe bugs are here with actions/upload-artifact, so the same bug is also reported to actions/upload-artifact: https://github.com/orgs/community/discussions/47721.https://github.com/actions/upload-artifact/issues/389

**Comment by viveklak** [comment]
This is pretty unfortunate. It was really easy to be able to download artifacts from the UI and also be able to delete them. Can we please have this back?

**Comment by OzdilOmer** [comment]
We've had the same issue in our monorepo after the v4 upgrade. the artifact seems to be uploaded successfully
![Capture -art](https://github.com/user-attachments/assets/9e337ee5-4889-477a-9083-caab36b2f29c)

But the folder isnt shown in the UI 
![Capture-UI](https://github.com/user-attachments/assets/14ba24ec-1914-4ab2-8702-592bdb957e2e)
